README.md

```diff
@@ -98,15 +98,131 @@ This release extends support for RHEL v8.5.
+- AMD Instinct™ MI250X
+
+#### Supported Workstation GPUs
+
+This ROCm release extends support to two workstation GPU products:
 
-**Note:** Radeon Pro v620 is not supported on SLES.
+- AMD Radeon™ PRO V620
+- AMD Radeon™ PRO W6800
+
+These features are verified to work by quality assurance (QA):
+
+- SRIOV virtualization support for AMD Radeon™ PRO V620
+- KVM Hypervisor (1VF support only) on Ubuntu Host OS with Ubuntu, CentOs, and RHEL Guest
+- Support for ROCm-SMI in an SRIOV environment. For more details, refer to [the ROCm SMI API documentation](https://rocmdocs.amd.com/en/latest/ROCm_System_Managment/ROCm-System-Managment.html).
+
+**Note:** AMD Radeon™ PRO V620 is not supported on SLES.
+
+#### Compatible Consumer GPUs
+
+All the supported workstation GPU products use the `gfx1030` instruction set architecture (ISA) implemented in the "Navi 21" silicon.
+The same silicon type implementing the same ISA is also used in some consumer GPU products, which *should* therefore be compatible with this ROCm release.
+However, ROCm was not validated against the following products before release:
+
+-   AMD Radeon™ RX 6800
+-   AMD Radeon™ RX 6800 XT
+-   AMD Radeon™ RX 6900 XT
+-   AMD Radeon™ RX 6950 XT
+
+#### Library Target Matrix
+
+In general, the libraries of this ROCm release can be used with GPU products which implement one of the instruction set architectures (ISA) listed in each of the libraries `TARGETS` variables.
+The current target GPU products with relevant instruction set architectures are:
+
+| ISA           | Family     | Chip            | Product                                           | RCCL | rocBLAS | rocFFT | rocPRIM | rocRAND | rocSolver | rocSparse | rocThrust | rocALUTION | rocWMMA |
+|:--------------|:-----------|:----------------|:--------------------------------------------------|:----:|:-------:|:------:|:-------:|:-------:|:---------:|:---------:|:---------:|:----------:|:-------:|
+| gfx803        | Fiji       | Fiji XT         | AMD Radeon™ R9 Fury                               |  ✅   |    ✅    |   ✅    |    ✅    |    ✅    |     ✅     |     ✅     |     ✅     |     ✅      |    ∅    |
+| gfx803        | Fiji       | Fiji XT         | AMD Radeon™ R9 Nano                               |  ✅   |    ✅    |   ✅    |    ✅    |    ✅    |     ✅     |     ✅     |     ✅     |     ✅      |    ∅    |
+| gfx803        | Fiji       | Fiji XT         | AMD Radeon™ R9 Fury X                             |  ✅   |    ✅    |   ✅    |    ✅    |    ✅    |     ✅     |     ✅     |     ✅     |     ✅      |    ∅    |
+| gfx803        | Fiji       | Fiji XT         | AMD Radeon™ Instinct MI8                          |  ✅   |    ✅    |   ✅    |    ✅    |    ✅    |     ✅     |     ✅     |     ✅     |     ✅      |    ∅    |
+| gfx803        | Fiji       | Capsaicin XT    | AMD FirePro S9300x2                               |  ✅   |    ✅    |   ✅    |    ✅    |    ✅    |     ✅     |     ✅     |     ✅     |     ✅      |    ∅    |
```

Comment:
If we're going to put trademark symbols on things, AMD puts (tm) on FirePro as well.

Reply:
Thanks for the in-depth review, @jlgreathouse! The trademarking seems to apply to the "Instinct" brand, too: https://www.amd.com/de/graphics/instinct-server-accelerators I addressed both issues in my latest commit.